_posts/2024-01-29-strimzicon2024-announcement.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Welcome StrimziCon 2024!"
+date: 2024-01-26
+author: paolo_patierno
+---
+
+We are very pleased to announce the first StrimziCon, scheduled later this year!
+This is going to be the first conference focused on our beloved Strimzi and Apache Kafka projects and it will be "virtual".
+It is organized and driven by the project maintainers in collaboration with the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+<!--more-->
+
+![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)
+
+## Why attend StrimziCon?
```

Comment:
You should use `###` for these to not make them the same level as the main title.